workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -105,6 +106,34 @@ public abstract class StatefulWorkflow<
     state: StateT
   ): StateT = state
 
+  @Composable
+  public open fun Rendering(
+    renderProps: PropsT,
+    renderState: StateT,
+    context: RenderContext,
+    hoistRendering: @Composable (rendering: RenderingT) -> Unit
+  ): Unit {
+    // By default call render() as a fallback.
+    hoistRendering(render(renderProps, renderState, context))
```

Comment:
I did this so I don't have to modify *every* workflow and test to get this to compile. However, at runtime the Workflow.Rendering gets resolved here rather than to the child function. 😢

Reply:
https://github.com/cashapp/molecule/issues/91